File: education/education/web_form/program_certificate_reqeust/program_certificate_reqeust.js
frappe.ready(function() {
	{% if is_new%}
	var student = `{{student_data}}`;
	console.log(student)
	student = JSON.parse(student)
	// frappe.web_form.fields_dict.course.df.options = courses;
	frappe.web_form.set_value(["first_name_arabic"], [student.first_name_arabic])
	frappe.web_form.set_value(["first_name_english"], [student.first_name])
	frappe.web_form.set_value(["middle_name_arabic"], [student.middle_name_arabic])
	frappe.web_form.set_value(["middle_name_english"], [student.middle_name])
	frappe.web_form.set_value(["last_name_arabic"], [student.last_name_arabic])
	frappe.web_form.set_value(["last_name_english"], [student.last_name])
	frappe.web_form.set_value(["nationality"], [student.nationality])
	frappe.web_form.set_value(["mobile_no"], [student.student_mobile_number])
	frappe.web_form.set_value(["email"], [student.student_email_id])
	frappe.web_form.set_value(["educational_level"], [student.educational_level])
	
	// frappe.web_form.set_value(["academic_term"], ["{{academic_term}}"])
	// frappe.web_form.refresh_fields([{fieldname: 'course'}])
	{% endif %}
	console.log(__('The information must be identical to an official document such as a passport'))
	$('div[data-fieldname="__section_1"] div.section-head').text('{{msg}}')
})

File: education/education/web_form/program_certificate_reqeust/program_certificate_reqeust.py
import frappe
import json
from frappe import _
def get_context(context):
	# do your magic here
	context.msg = _('The information must match the passport. If you do not have a passport, contact us')
	if context.is_new:
		student = frappe.db.get_value("Student", {"user": frappe.session.user}, ["name", 
									   "first_name", "middle_name", "last_name",
									   "first_name_arabic", "middle_name_arabic", "last_name_arabic","educational_level",
									   "student_mobile_number", "student_email_id", "nationality"], as_dict=True)
		if not student: return
		context.student_data = json.dumps(student)
